src/utils/types.ts

```diff
@@ -40,6 +41,12 @@ export type ElementAttributes = {
     lastModifiedBy: string;
     children: any[];
     parentUuid: null | UUID;
+    specificMetadata: {
```

Comment:
This is different for each element type

Reply:
i suggest to put it as 'object' and create another PR to specify a type for each element like 'StudySpecificMetaData', 'FilterSpecificMetadata' ...